pkg/snmp/snmpintegration/config.go

```diff
@@ -7,8 +7,9 @@ package snmpintegration
 
 // InterfaceConfig interface related configs (e.g. interface speed override)
 type InterfaceConfig struct {
-	MatchField string `mapstructure:"match_field" yaml:"match_field" json:"match_field"` // e.g. name, index
-	MatchValue string `mapstructure:"match_value" yaml:"match_value" json:"match_value"` // e.g. eth0 (name), 10 (index)
-	InSpeed    uint64 `mapstructure:"in_speed" yaml:"in_speed" json:"in_speed"`          // inbound speed override in bps
-	OutSpeed   uint64 `mapstructure:"out_speed" yaml:"out_speed" json:"out_speed"`       // outbound speed override in bps
+	MatchField string   `mapstructure:"match_field" yaml:"match_field" json:"match_field"` // e.g. name, index
+	MatchValue string   `mapstructure:"match_value" yaml:"match_value" json:"match_value"` // e.g. eth0 (name), 10 (index)
+	InSpeed    uint64   `mapstructure:"in_speed" yaml:"in_speed" json:"in_speed"`          // inbound speed override in bps
+	OutSpeed   uint64   `mapstructure:"out_speed" yaml:"out_speed" json:"out_speed"`       // outbound speed override in bps
+	Tags       []string `mapstructure:"tags" yaml:"tags" json:"tags"`                      // interface tags
```

Comment:
I think we need a reno changelog for this feature.

You can create changelog like this: https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno

Reply:
Added one, thanks!